src/shared/ui/pickerPrompter.ts

```diff
@@ -137,7 +137,7 @@ export function createQuickPick<T>(
 
     const prompter =
         mergedOptions.filterBoxInputSettings !== undefined
-            ? new FilterBoxQuickPickPrompter<T>(picker, mergedOptions.filterBoxInputSettings)
+            ? new FilterBoxQuickPickPrompter<T>(picker, mergedOptions)
             : new QuickPickPrompter<T>(picker, mergedOptions)
```

Comment:
@JadenSimon was this intentional? In order to show `noItemsFoundItem` simultaneously with `filterBoxInputSettings` I changed `FilterBoxQuickPickPrompter` to pass the full `options` to its baseclass.